pom.xml

```diff
@@ -53,9 +53,9 @@
         <dependency>
             <groupId>org.owasp</groupId>
             <artifactId>zaproxy-client-api</artifactId>
-            <version>2.4.3</version>
+            <version>2.6</version>
             <scope>system</scope>
-            <systemPath>${project.basedir}/lib/zap-api-2.4-v9.jar</systemPath>
+            <systemPath>${project.basedir}/lib/zap-api-1.2.0.jar</systemPath>
```

Comment:
The library is (now) available in Maven Central, might worth using that instead.